orchestrate.py

```diff
@@ -174,6 +174,35 @@ def post(self):
     def pool(self):
         return self.settings['pool']
 
+class AdminHandler(RequestHandler):
+    def prepare(self):
+        '''
+        Responds with a 401 error if the configured admin auth token is not 
+        present in the request headers.
+        '''
+        if self.admin_token:
+            client_token = self.request.headers.get('Authorization')
+            if client_token != 'token %s' % self.admin_token:
+                app_log.warn('rejecting admin request with token %s', client_token)
+                app_log.info('admin token: %s', self.admin_token)
```

Comment:
Capitalized the first character of the "rejecting ..." warning. Removed the info which was leftover while I was debugging. (Definitely don't want to log the admin token!)